### ISSUE TITLE
Add Social Icons

### ISSUE THREAD

**Issue by alistairholmes** [DigitalNomadJobs/digitalnomadjobs.github.io#4]
Needs issue #3  to be completed first. 

Add icons for [Instagram](https://www.instagram.com/digitalnomadjobs/) and Twitter that will link to social pages. This will be just below the Google Play download button as seen in the Inspiration website.

Please have a look at the Inspiration Section and follow the Contribution guidelines in README.md

**Comment by dharmadev108** [comment]
I'll do it: is that ok?

**Comment by alistairholmes** [comment]
Sure but issues #2  and #3 will have to be completed first.

**Comment by dharmadev108** [comment]
Ok, I'll start with #2 then ;)

**Comment by dharmadev108** [comment]
Just to get this straight: at current state, the whole webpage is missing and you need someone to code the whole thing, right?  
Do you have any graphic design element already at hand to complete this task? Or do you need a contribution from that point of view as well? 
Thanks :)

**Comment by alistairholmes** [comment]
@dharmadev108 yes, need a whole webpage constructed, but I've broken it up into different issues, Please see the [Inspiration Section](https://github.com/DigitalNomadJobs/digitalnomadjobs.github.io#inspiration) for details on the UI design

**Comment by dharmadev108** [comment]
ok cool, i can do it. just one thing: could you please provide some screenshots of your app? i don't own an android device so it would really be of help for creating the image on the left.

**Comment by alistairholmes** [comment]
#5 Added the screenshots for right hand side of the site.
